<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>228</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>228</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(185, 28, 28);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>240</width>
      <height>151</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>170</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 128);
color: rgb(0, 0, 128);
 border-radius: 10px;</string>
    </property>
    <property name="midLineWidth">
     <number>6</number>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::WidgetWidth</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:14pt; font-weight:700; color:#000080;&quot;&gt;Thank You for Registering!&lt;br /&gt;We're glad to have you on board. Your willingness to help can make a life-saving difference. Together, we can build a stronger, healthier community—one donation at a time.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Общий стиль для всего окна */
QWidget {
    background-color: #b30000; /* насыщенный красный */
    font-family: 'Segoe UI', sans-serif;
    color: white;
}

/* Заголовки */
QLabel#titleLabel {
    font-size: 24px;
    font-weight: bold;
    color: white;
}

/* Подзаголовки и текст */
QLabel {
    font-size: 14px;
    color: white;
}

/* Поля ввода */
QLineEdit {
    background-color: white;
    border: 2px solid #ff4d4d;
    border-radius: 10px;
    padding: 6px;
    color: #333;
    font-size: 14px;
}

/* Фокус на поле */
QLineEdit:focus {
    border: 2px solid #ff1a1a;
}

/* Кнопки */
QPushButton {
    background-color: #b30000;
    color: white;
    font-weight: bold;
    font-size: 14px;
    border-radius: 12px;
    padding: 8px 16px;
    border: 2px solid white;
}

QPushButton:hover {
    background-color: #ffcccc;
    color: #800000;
}

QPushButton:pressed {
    background-color: #ffe6e6;
    border: 2px solid #ff9999;
}

/* Ошибки или уведомления */
QLabel#errorLabel {
    color: #ffcccc;
    font-size: 12px;
}

/* Кнопки внизу: Sign In / Sign Up переключение */
QPushButton#switchButton {
    background: transparent;
    color: white;
    font-size: 12px;
    text-decoration: underline;
    border: none;
}

QPushButton#switchButton:hover {
    color: #ff9999;
}</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>278</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
